src/components/common/MainMenu.js

```diff
@@ -46,6 +46,7 @@ const MainMenu = () => {
 
   return (
     <GraaspMainMenu id={MAIN_MENU_ID}>
+      <div style={{ height: 10 }} />
```

Comment:
Is this to push the tree view bellow the app bar ? I think there is a component specifically for this.

Reply:
It's actually just for the little margin... I don't have control on the inside and the outside of these two components... But I'll add a todo.